admin_guide/tcp_ingress_external_ports.adoc

```diff
@@ -0,0 +1,110 @@
+[[admin-guide-unique-external-ips-ingress-traffic]]
+= Assigning Unique External IPs for Ingress Traffic
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Cluster administrators can assign a unique external IP address to a service. If
+routed correctly, external traffic can reach that service's endpoints via any
+TCP/UDP port the service exposes. This can be simpler than trying to manage the
+port space of a limited number of shared IP addresses, and manually assigning
+external IP addresses to services.
+
+There is support for both automatic and manual assignment of IP addresses, and
+each address is guaranteed to be assigned to a maximum of one service. This
+ensures that each service can simply expose its chosen ports regardless of the
+ports exposed by other services.
+
+[[unique-external-ips-ingress-traffic-restrictions]]
+== Restrictions
+
+To use an `*ExternalIP*`, you can:
+
+- Select an IP address from the `*ExternalIPNetworkCIDRs*` range. 
+- Have an IP address assigned from a pool. In this case, {product-title} implements a non-cloud version of the LoadBalancer service type and assigns IP addresses to the services.
++
+[CAUTION]
+====
+You must ensure that the IP address pool you assign terminates at one or more
+nodes in your cluster. You can use the existing `*oadm ipfailover*` to ensure
+that the external IPs are highly available.
```

Comment:
Is there a link we could put to how to use `oadm ipfailover` from the router HA docs?